portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,32 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/index")
 public class DataServlet extends HttpServlet {
-
+  private List<String> intros;
+  private int count;
   @Override
+  public void init() {
+    intros = new ArrayList<>();
+    intros.add("Hey! ");
+    intros.add("Hello! ");
+    intros.add("What's up! ");
```

Comment:
Instead of adding the space at the end of each array entry, I might consider adding the space when you actually display it.